src/test/util/test_event.rs

```diff
@@ -36,9 +36,19 @@ impl Matchable for TestEvent {
                     command: expected_command,
                 },
             ) => {
-                actual_command_name == expected_command_name
```

Comment:
Why did this need to be changed? It looks like the newer version could possibly have some false positives.

Reply:
Some of the command monitoring expectations in the tests don't include all of the fields (e.g. [this one](https://github.com/mongodb/specifications/blob/8b1d6e9ff4a7dfd2a47d3fcf6e0ee6b2d98291a7/source/retryable-reads/tests/aggregate.json#L70) is missing `command_name`), so they need to be options to ignore the missing fields. The method on `EventClient` that returns all of the events to compare against the expectations already filters out any events that don't match the name of the operation though, so the comparison still occurs.

Reply:
The command monitoring tests are pretty old, so they don't use the unified/v2 test format. Because of that, I wouldn't make any changes in your new runner to accommodate them. Instead, just leave command monitoring as having a separate implementation so the new one can be left unhampered. If the command monitoring tests can borrow some stuff without having to make changes to the runner for them, then by all means share code though.